src/gui/optionsdialog.cpp

```diff
@@ -44,6 +44,7 @@
 #include <QMessageBox>
 #include <QSystemTrayIcon>
 #include <QTranslator>
+#include <qstylehints.h>
```

Comment:
```suggestion
#include <QStyleHints>
```

Reply:
Hmm... I didn't include it manually. And I even doubt it is needed here.

Reply:
```suggestion
```